clang-tools-extra/clang-move/tool/ClangMove.cpp

```diff
@@ -199,7 +199,7 @@ int main(int argc, const char **argv) {
       for (auto I = Files.begin(), E = Files.end(); I != E; ++I) {
         OS << "  {\n";
         OS << "    \"FilePath\": \"" << *I << "\",\n";
-        const auto Entry = FileMgr.getFile(*I);
+        const auto Entry = FileMgr.getOptionalFileRef(*I);
```

Comment:
Having `*Entry` on the next line without checking isn't great, but obviously not a new issue in this PR you need to fix.